config/elasticsearch.yml

```diff
@@ -88,58 +84,6 @@ cluster.name: logger
 # to disable it, set the following:
 # node.max_local_storage_nodes: 1
 
-
-#################################### Index ####################################
```

Comment:
Since Elasticsearch 5.x index level settings can NOT be set on the nodes
configuration like the elasticsearch.yaml